app/src/main/java/de/hbch/traewelling/ui/checkIn/EditCheckInFragment.kt

```diff
@@ -70,6 +85,28 @@ class EditCheckInFragment : AbstractCheckInFragment() {
                 )
                 alertBottomSheet.show(parentFragmentManager, AlertBottomSheet.TAG)
             }
-        })
+        }
+        if (!args.replace) {
+            TraewellingApi.checkInService.updateCheckIn(
+                args.statusId, UpdateStatusRequest(
+                    model.message.value,
+                    model.statusBusiness.value ?: error("Invalid data"),
+                    model.statusVisibility.value ?: error("Invalid data")
+                )
+            ).enqueue(callback)
+        } else {
+            checkInViewModel.startStationId = args.startStationId
+            checkInViewModel.destinationStationId = args.destinationId
+            checkInViewModel.departureTime = args.departureTime
+            TraewellingApi.checkInService.updateCheckIn(
+                args.statusId, UpdateStatusRequest(
+                    model.message.value,
+                    model.statusBusiness.value ?: error("Invalid data"),
+                    model.statusVisibility.value ?: error("Invalid data"),
+                    args.destinationId,
+                    args.departureTime
+                )
```

Comment:
Is the .enqueue() call missing here?

Reply:
yes didn't get to test the API call yet due to waiting for that pr